src/main/java/com/autotune/analyzer/serviceObjects/CreateExperimentAPIObject.java

```diff
@@ -0,0 +1,206 @@
+    @SerializedName(KruizeConstants.JSONKeys.CLUSTER_NAME)
+    private String clusterName;
+    @SerializedName(KruizeConstants.JSONKeys.PERFORMANCE_PROFILE)
+    private String performanceProfile;
+    @SerializedName("slo")
+    private SloInfo sloInfo;
+    @SerializedName(KruizeConstants.JSONKeys.MODE)
+    private String mode;
+    @SerializedName(KruizeConstants.JSONKeys.TARGET_CLUSTER)
+    private String targetCluster;
+    @SerializedName(KruizeConstants.JSONKeys.KUBERNETES_OBJECTS)
+    private List<KubernetesObject> kubernetesObjects;
+    @SerializedName(KruizeConstants.JSONKeys.TRIAL_SETTINGS)
+    private TrialSettings trialSettings;
+    @SerializedName(KruizeConstants.JSONKeys.RECOMMENDATION_SETTINGS)
+    private RecommendationSettings recommendationSettings;
+
+    public String getClusterName() {
+        return clusterName;
+    }
+
+    public void setClusterName(String clusterName) {
+        this.clusterName = clusterName;
+    }
+
+    public String getPerformanceProfile() {
+        return performanceProfile;
+    }
+
+    public void setPerformanceProfile(String performanceProfile) {
+        this.performanceProfile = performanceProfile;
+    }
+
+    public String getMode() {
+        return mode;
+    }
+
+    public void setMode(String mode) {
+        this.mode = mode;
+    }
+
+    public String getTargetCluster() {
+        return targetCluster;
+    }
+
+    public void setTargetCluster(String targetCluster) {
+        this.targetCluster = targetCluster;
+    }
+
+    public List<KubernetesObject> getKubernetesObjects() {
+        return kubernetesObjects;
+    }
+
+    public void setKubernetesObjects(List<KubernetesObject> kubernetesObjects) {
+        this.kubernetesObjects = kubernetesObjects;
+    }
+
+    public TrialSettings getTrialSettings() {
+        return trialSettings;
+    }
+
+    public void setTrialSettings(TrialSettings trialSettings) {
+        this.trialSettings = trialSettings;
+    }
+
+    public RecommendationSettings getRecommendationSettings() {
+        return recommendationSettings;
+    }
+
+    public void setRecommendationSettings(RecommendationSettings recommendationSettings) {
+        this.recommendationSettings = recommendationSettings;
+    }
+
+    public SloInfo getSloInfo() {
+        return sloInfo;
+    }
+
+    public void setSloInfo(SloInfo sloInfo) {
+        this.sloInfo = sloInfo;
+    }
+
+    @Override
+    public String toString() {
+        return "CreateExperimentSO{" +
+                "clusterName='" + clusterName + '\'' +
+                ", performanceProfile='" + performanceProfile + '\'' +
+                ", mode='" + mode + '\'' +
+                ", targetCluster='" + targetCluster + '\'' +
+                ", kubernetesObjects=" + kubernetesObjects +
+                ", trialSettings=" + trialSettings +
+                ", recommendationSettings=" + recommendationSettings +
+                '}';
+    }
+}
+class KubernetesObject {
```

Comment:
Lets call this `KubernetesAPIObject`. Please add a javadoc saying this is simulating the `k8sObject`

Reply:
Done